tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -323,6 +323,17 @@ jobs:
             customCommand: 'install'
             customRegistry: 'useNpmrc'
 
+      - task: Cache@2
+        displayName: Cache compat versions install location
+        timeoutInMinutes: 3
+        continueOnError: true
+        inputs:
+          key: '"compat-version-installs" | "$(Agent.OS)" | "${{ parameters.testCommand }}" | "${{ variant.name }}"'
+          path: ${{ parameters.testWorkspace }}/node_modules/@fluid-private/test-version-utils/node_modules/.legacy/
+          restoreKeys: |
+            "compat-version-installs" | "$(Agent.OS)" | "${{ parameters.testCommand }}"
+            "compat-version-installs" | "$(Agent.OS)"
```

Comment:
Are you sure you want to restore a cache from a different testCommand?

Reply:
Hmmm I think it's fine but might be better to remove it... I'm not sure exactly _what_ from an older version is installed, so I guess we could end up with some bits that testCommand A installed, but they're not exactly the ones that testCommand B wants? And since `insalled.json` would already have that version, there'd be no attempt to install the other pieces. Seems unlikely that our older-version installs work like that, but to be safe I'll get rid of the additional restore keys.